compiler/src/iree/compiler/Codegen/Common/GenericVectorization.cpp

```diff
@@ -153,7 +153,8 @@ void GenericVectorizationPass::runOnOperation() {
   funcOp.walk([&](Operation *op) {
     if (isa<linalg::LinalgOp>(op))
       candidates.push_back(op);
-    if (vectorizePadding && enableVectorMasking && isa<tensor::PadOp>(op))
+    if (vectorizePadding && enableVectorMasking &&
+        (isa<tensor::PadOp>(op) || isa<tensor::UnPackOp>(op)))
```

Comment:
This is not related to the PR, can we revert it?

Reply:
You will need it when we enable the direct vectorization for unpack ops. This is unrelated to the PR itself, so I suggest to remove it from the PR. This is not working until we have the upstream support. I prefer adding the change when it is available and with an update on [the pipeline test](https://github.com/openxla/iree/blob/bd2c92dbb3d2109cd624fa18e75b9bf3caaa4ae5/compiler/src/iree/compiler/Codegen/LLVMCPU/test/pipeline_pack_unpack_tests.mlir#L50-L130).

Reply:
Ok that makes sense to me. I removed it.